templates/todo/web/react-fluent/package.json

```diff
@@ -16,10 +16,12 @@
     "envconfig": "node entrypoint.js -e .env -o ./public/env-config.js",
     "prestart": "npm run envconfig",
     "start": "if-node-version '>= 17' && react-scripts --openssl-legacy-provider start || react-scripts start",
-    "prebuild": "npm run envconfig",
-    "build": "if-node-version '>= 17' && react-scripts --openssl-legacy-provider build || react-scripts build",
+    "prebuild": "npm run lint",
+    "build": "tsc -b .",
```

Comment:
How idiomatic is it for a react app to have `build` vs `package` separately? Just curious if we're doing something that users might not expect?

Reply:
I think in previous projects I've worked on, I'd expect to perform `npm run build` and then my app is ready to run locally. Does this still work for that scenario?

Reply:
Over time we will need to expose npm configuration in `azure.yaml` to allow to override the defaults scripts that are run.